doc/reference/annotations.rst

```diff
@@ -406,20 +406,20 @@ Available Types:
 |                                                            | string (''). DeserializeFormats can either be a  |
 |                                                            | string or an array of string.                    |
 +------------------------------------------------------------+--------------------------------------------------+
-| DateTimeInterface                                          | PHP's DateTimeImmutable object (default format*/ |
-|                                                            | timezone).                                       |
+| DateTimeInterface                                          | PHP's DateTimeInterface interface (default       |
+|                                                            | format*/timezone).                               |
```

Comment:
Could we add information to which class it will be deserialised same as in Iterators, please?